Marlin/src/gcode/control/M3-M5.cpp

```diff
@@ -126,6 +133,11 @@ void GcodeSuite::M5() {
     // Non-inline, standard case
     cutter.inline_disable(); // Prevent future blocks re-setting the power
   #endif
+
+  #if ENABLED(SPINDLE_SERVO)
+    servo[SPINDLE_SERVO_ID].move(parser.intval('S', SPINDLE_SERVO_MIN));
```

Comment:
The M3-M5 implementations make me ask two questions:

1. Should the actual servo setting be moved into the `LaserSpindle` (`cutter`) class?

2. If not, are these functions doing things with the `cutter` variable which should now be skipped?